shared/featureSwitches.ts

```diff
@@ -25,4 +25,5 @@ export const initFeatureSwitchUrlParamOverride = () => {
 export const featureSwitches: Record<string, boolean> = {
 	exampleFeature: false,
 	cancellationProductSwitch: false,
+	accountOverviewNewLayout: false,
```

Comment:
The PR says the feature flag is `accountOverviewRefresh`

Reply:
Thanks! 🙏 I changed this yesterday, but forgot to update the description.